cmd/evm/runner.go

```diff
@@ -301,7 +301,7 @@ func runCmd(ctx *cli.Context) error {
 			os.Exit(1)
 		}
 		if err := pprof.WriteHeapProfile(f); err != nil {
-			fmt.Println("could not write memory profile: ", err)
+			fmt.Println("could not write memory profile")
```

Comment:
Above changes are fine, Any reason for this change ?
@marcello33

Reply:
This is a funny one. `snyk` detects this print as a leak of vulnerable info. 
We could revert this and add the alert to the list of ignored ones.

Reply:
Yeah let's do that, Because I don't want this small change to create merge conflict with future upstream changes.
I don't think this is a vulnerable info 😄

Reply:
Sure, I agree. Solved here [db21bfe5be88e60860a8af580a6641f919f4ed90](https://github.com/maticnetwork/erigon/pull/8/commits/db21bfe5be88e60860a8af580a6641f919f4ed90)